docs/releases/3.2.txt

```diff
@@ -348,6 +348,9 @@ Models
 
 * Added the :class:`~django.db.models.functions.Random` database function.
 
+* Expressions now support transform references, allowing ``__`` lookup syntax
+  in aggregation, annotations, F expressions, etc.
```

Comment:
I'm not especially happy with the phrasing here. Suggestions welcome!

Reply:
Maybe:
```
* :ref:`aggregation-functions`, :class:`F() <django.db.models.F>`, and
  :class:`OuterRef() <django.db.models.OuterRef>` expressions now allow using
  transforms and :ref:`lookups that span relationships
  <lookups-that-span-relationships>`. See
  :ref:`using-lookups-transforms-in-expressions` for details.
```